src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -939,6 +939,13 @@
     <Field type="ulong" contentType="address" name="topAddress" label="Top Address" description="Ending address of the module, if available" />
   </Event>
 
+  <Event name="NativeLibraryLoad" category="Java Virtual Machine, Runtime" label="Native Library Load Operation" thread="false" stackTrace="true" startTime="true"
```

Comment:
Hi Erik, I will adjust. I will use errorMessage because on some platforms we have already coding to describe shared lib load error situations in more detail (see for example os_linux.cpp).